### ISSUE TITLE
ActionSheet style on iPad does not have OK button

### ISSUE THREAD

**Issue by hstdt** [dillidon/Alerts-Pickers#22]
![image](https://user-images.githubusercontent.com/10215098/35893633-f6b9c622-0be9-11e8-8724-68773f4a4f91.png)

**Comment by dillidon** [comment]
@hstdt Yes, on iPad actionButton with `.cancel` style doesn't appear. Use `.default` or `.destructive` instead of `.cancel`.
